src/operationDefinitions/index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import { OperationDefinitionImplementation } from './types';
+import { USCoreDocRef } from './USCoreDocRef';
+import { OperationDefinitionRegistry } from './OperationDefinitionRegistry';
+import ConfigHandler from '../configHandler';
+
+export const initializeOperationRegistry = (configHandler: ConfigHandler) => {
+    const { compiledImplementationGuides } = configHandler.config.profile;
+    const operations: OperationDefinitionImplementation[] = [];
+
+    if (
```

Comment:
Minor comment, thoughts?

```suggestion
   // Add US Core $docref operation if found in the compiled implementation guide. Note: this is an example and can be removed if not desired
    if (
```

Reply:
I added some comments here.
 
Yeah, I do think that customers are likely to modify this particular file to change how operations are enabled/disabled.

Right now customers that don't want to modify source code have the following options:
- Do not use US Core, $docref is disabled
- Add US Core IG, $docref is enabled
- Add US Core, but delete the $docref OperationDefinition file before compiling, $docref is disabled (this one is a bit awkward, but I believe that this is the least common use case)